### PR TITLE
update profiling for 12_1_0_pre5 to the webpage

### PR DESCRIPTION
**Pull request by xoqhdgh1002**

### CODE REVIEW

File: web/index.html [hunk 1]
```diff
@@ -66,17 +60,6 @@ <h3>Summary of CMSSW_11_0_X(PAT_MINIAOD)(29234.21) <font size="2em"><a target='_
 			<li><strong>step3_AOD(RECO):</strong>
 			<a href="http://cms-reco-profiling.web.cern.ch/cms-reco-profiling/results/Time_Mem_Summary/CMSSW_11_0_0/slc7_amd64_gcc820/20634.21/step3.txt" title="getTimeMemSummary">[getTimeMemSummary]</a>
 			<ul>
-			<li>Comparison:
```
Comment (jpata) [inline web/index.html:69]: did you remove this intentionally?

File: web/index.html [hunk 2]
```diff
@@ -85,25 +68,13 @@ <h3>Summary of CMSSW_11_0_X(PAT_MINIAOD)(29234.21) <font size="2em"><a target='_
 			<li><strong>step4_MiniAOD(PAT):</strong>
 			<a href="http://cms-reco-profiling.web.cern.ch/cms-reco-profiling/results/Time_Mem_Summary/CMSSW_11_0_0/slc7_amd64_gcc820/20634.21/step4.txt" title="getTimeMemSummary">[getTimeMemSummary]</a>
 			<ul>
-			<li>Comparison:
```
Comment (jpata) [inline web/index.html:88]: did you remove this intentionally?

File: web/index.html [hunk 3]
```diff
@@ -161,25 +132,13 @@ <h3>Summary of CMSSW_11_1_X(PAT_MINIAOD)(20634.21) <font size="2em"><a target='_
 				<a href="http://cms-reco-profiling.web.cern.ch/cms-reco-profiling/cgi-bin/igprof-navigator/releases/CMSSW_11_1_0/20634.21/step4//mem_live.49" title="MEM time check">[MEM Profiler igprof.49]</a>
 				<a href="http://cms-reco-profiling.web.cern.ch/cms-reco-profiling/results/RES/CMSSW_11_1_0/slc7_amd64_gcc820/20634.21/step4_mem.res" title="txtLink">[txtLink]</a>
 				<ul>
-			<li>Comparison:
```
Comment (jpata) [inline web/index.html:164]: did you remove this intentionally?

File: web/index.html [hunk 4]
```diff
@@ -1228,16 +1187,10 @@ <h3>Summary of CMSSW_11_2_X(PAT_MINIAOD)(23434.21) <font size="2em"><a target='_
 </ul>
 <hr size="3" noshade>
         <h2>CMSSW_11_3_X <font size="4em"></font></h2>
-
 <span style=" font: normal bold 1.0em Georgia, serif ; color: navy;">
         <h3>Summary of CMSSW_11_3_X(RECO_AOD)(23434.21) <font size="2em"><a target='_blank' href="http://cms-reco-profiling.web.cern.ch/cms-reco-profiling/results/summary_plot_html/CMSSW_11_3_step3_23434.21.html" title="shortSummary">[Short Summary Time and Memory(AOD)]</a></font></h3>
         <h3>Summary of CMSSW_11_3_X(PAT_MINIAOD)(23434.21) <font size="2em"><a target='_blank' href="http://cms-reco-profiling.web.cern.ch/cms-reco-profiling/results/summary_plot_html/CMSSW_11_3_step4_23434.21.html" title="shortSummary">[Short Summary Time and Memory(PAT)]</a></font></h3>
 </span>
-<hr size="2" noshade>
```
Comment (jpata) [inline web/index.html:1236]: did you remove this intentionally?

File: web/index.html [hunk 5]
```diff
@@ -4048,3 +3986,228 @@ <h3>Summary of CMSSW_12_1_X(PAT_MINIAOD)(34834.21) <font size="2em"><a target='_
...
+		<ul>
+
+			<li><strong>step3_AOD(RECO):</strong>
+			<a href="http://cms-reco-profiling.web.cern.ch/cms-reco-profiling/results/Time_Mem_Summary/CMSSW_12_1_0_pre5/slc7_amd64_gcc900/11834.21/step3.txt" title="getTimeMemSummary">[getTimeMemSummary]</a>
+			
+				<a href="http://cms-reco-profiling.web.cern.ch/cms-reco-profiling/cgi-bin/igprof-navigator/releases/CMSSW_12_1_0_pre5/11834.21/step3//cpu_endjob" title="CPU time check">[CPU Profiler igprof.]</a>
+				<a href="http://cms-reco-profiling.web.cern.ch/cms-reco-profiling/results/RES/CMSSW_12_1_0_pre5/slc7_amd64_gcc900/11834.21/step3_cpu.res" title="txtLink">[txtLink]</a>
+				
+				<a href="http://cms-reco-profiling.web.cern.ch/cms-reco-profiling/cgi-bin/igprof-navigator/releases/CMSSW_12_1_0_pre5/11834.21/step3//mem_live.1" title="MEM time check">[MEM Profiler igprof.1]</a>
+				<a href="http://cms-reco-profiling.web.cern.ch/cms-reco-profiling/results/RES/CMSSW_12_1_0_pre5/slc7_amd64_gcc900/11834.21/step3_mem.res" title="txtLink">[txtLink]</a>
```
Comment (jpata) [inline web/index.html:4001]: stepN_mem.res is giving 404/not found. Moreover, it is the same URL for all (.1, .200/.50, .399/.99).

**Review by jpata — COMMENTED** [review]
Thanks for the update! I left a few comments inline, please check.

File: web/index.html (hunk 1 above)
Comment (xoqhdgh1002) [reply web/index.html:69]: Yes, This part does not need. Compare data doesn't exist.

File: web/index.html (hunk 2 above)
Comment (xoqhdgh1002) [reply web/index.html:88]: Yes, This part does not need. Compare data doesn't exist.

File: web/index.html (hunk 3 above)
Comment (xoqhdgh1002) [reply web/index.html:164]: Yes, This part does not need. Compare data doesn't exist.

File: web/index.html (hunk 4 above)
Comment (xoqhdgh1002) [reply web/index.html:1236]: I combined links for same version.

File: web/index.html (hunk 5 above)
Comment (xoqhdgh1002) [reply web/index.html:4001]: I think this typo occurred on the previous web page.
I will fix that and will update PR again.

**Comment by jpata** [comment]
Thank you, looks good! I will merge it and deploy it.

**Pull request merged**